README.md

```diff
@@ -36,7 +36,8 @@ The first step to create the plugin is to name it. To do it, you have to edit 3
 - Edit `config.js` to change the name of your extension.
 - Edit `assets/index.json` and change the "name" entry with the name of your plugin. (here you can customize dependencies, if needed)
 - Edit `localConfig.json` replacing "SampleExtension", in `plugins/desktop` section, with the name of your Extension (for running local development)
-- Edit `package.json` and change `name` with a unique name, or follow the alternative solution as for #12
+
+Edit the package.json's name is not needed anymore because the build process uses the name of the extension instead. Anyway it is recommended to use a unique `name` in your `package.json` for each new extension.
```

Comment:
```suggestion
- *[only for version <= 2020.01.xx]* Edit the `package.json` `name` is not needed anymore because the build process uses the name of the extension instead. 

> **note** Edit the `name` in `package.json` is not needed strictly needed from version 2021.02.xx. Anyway it is a good practice to choose a unique `name` in your `package.json` for a new npm project, in general.
```